sdk/communication/tests.yml

```diff
@@ -23,11 +23,10 @@ stages:
         ServiceDirectory: communication
         DeployArmTemplate: true
         TestSamples: true
-        EnvVars:
-          AZURE_TEST_RUN_LIVE: 'true'
```

Comment:
Can you get rid of the `AZURE_TEST_RUN_LIVE` as well?

Reply:
I'll add that to the python config since this is python only :)

Reply:
also need to add AZURE_COMMUNICATION_SERVICE_CONNECTION_STRING